modules/lang-painless/src/test/resources/rest-api-spec/test/painless/80_script_score.yml

```diff
@@ -496,3 +496,53 @@
     - match: { hits.total: 2 }
     - match: { hits.hits.0._id : "2" }
     - match: { hits.hits.1._id : "1" }
+
+
+---
+"Script Score With Highlight":
+  - skip:
+      version: " - 7.4.09"
```

Comment:
I think this should just be ` - 7.4.0`?

Reply:
@jtibshirani thanks, corrected in the last commit